docs/source/guide/zne-3-options.myst

```diff
@@ -212,6 +216,88 @@ folding method. Fidelity values must be between zero and one.
 
 +++
 
+### Identity Scaling
+
+Mitiq has only one noise scaling function for the *identity scaling* technique. The goal of this technique is to insert
+layers of identity gates such that they act evenly across all qubits in the circuit. A unitary $G$ is mapped as follows after
+a single identity layer is inserted :
+
+$$G \longrightarrow G I.$$
+
+Here, $G$ is one layer of gates in the input circuit where the gates in each layer are supposed to act at the same time instance.
+
+The Mitiq function for *identity insertion scaling* is {func}`.insert_id_layers()`. Depending on the `scale_factor`, the number of
+identity layers are inserted in such a way that the new scale factor is exactly the desired `scale_factor` or is very close to it. 
+
+
+#### The difference between integer scale factors and real scale factors
+
+When the scale factor is 1, no identity layers are inserted and circuit depth remains unchanged. For some scale factor greater than 1, 
+there will be some layers that are inserted uniformly after each moment and some inserted at random to get the desired scale factor. 
+
+Consider an input circuit of depth $d$.
+
+Let $\lambda$ be the desired scale factor, $n$ be the uniformly inserted identity layers and $s$ be the randomly inserted identity layers. 
+
+The circuit depth of the returned identity scaled circuit is $\lambda d = d(1+n) + s$. 
+
+Thus, an expression for the scale factor can be derived by comparing the new circuit depth to the initial circuit depth. 
+
+$$\lambda = 1 + n + \frac{s}{d} = 1 + \frac{dn+s}{d} = 1+ \frac{k}{d}.$$
+
+When $\lambda$ is an integer, $\lambda = 1+x$ where $x$ is some integer less than $\lambda$. If the scale factor is an integer, identity
+layers are inserted uniformly after each moment in the input circuit because $n = x$.
```

Comment:
```suggestion
##### Example

Consider an input circuit of depth $d$.
Let $\lambda$ be the desired scale factor, $n$ be the number of uniformly inserted identity layers and $s$ be the number of randomly inserted identity layers. 
The scaled circuit depth is then approximated as $\lambda d \approx d(1+n) + s$. 
```

The rest of the derivation didn't make sense to me as the scale factor is specified ahead of time, so I'm unsure why we are "deriving" it.